extensions/WEBGL_compressed_texture_s3tc/extension.xml

```diff
@@ -24,6 +24,14 @@
         <code>COMPRESSED_RGBA_S3TC_DXT1_EXT</code>, <code>COMPRESSED_RGBA_S3TC_DXT3_EXT</code>, and
         <code>COMPRESSED_RGBA_S3TC_DXT5_EXT</code> may be passed to
         the <code>compressedTexImage2D</code> and <code>compressedTexSubImage2D</code> entry points.
+
+        <div class="note">
+          Some platforms do not have native support for <code>COMPRESSED_RGB_S3TC_DXT1_EXT</code> format,
+          so WebGL implementations may map it to <code>COMPRESSED_RGBA_S3TC_DXT1_EXT</code>.
```

Comment:
Could you please describe the consequences here, like you did in the comment in the test case below?

"This means the alpha channel is not guaranteed to be opaque (0xFF); it may be transparent (0x0) in some implementations."